### PR TITLE
data/selinux: add missing rule to fix selinux-clean spread tests

### PR DESCRIPTION
**Pull request by Meulengracht**

### CODE REVIEW

**Review by sergiocazzolato — COMMENTED** [review]
thanks

**Review by sergiocazzolato — APPROVED** [review]

**Review by olivercalder — COMMENTED** [review]
Looks good. Assuming this permission is required by a particular interface, it would be have a way to know which interface requires this permission, and further, to only grant it when that particular interface is used.

File: data/selinux/snappy.te [hunk 1]
```diff
@@ -494,6 +494,7 @@ admin_pattern(snappy_mount_t, snappy_tmp_t)
 # not happen when snap-confine is executed by unconfined_t, should we still end
 # up with user_tmp_t allow acting on it
 allow snappy_mount_t user_tmp_t:dir { mounton rmdir };
+allow snappy_mount_t etc_t:dir mounton;
```
[the remark this answers is not in the document]
Comment (Meulengracht) [reply data/selinux/snappy.te]: Good point, moved it, and no additional statements is neccessary for this rule

**Review by ernestl — APPROVED** [review]

File: data/selinux/snappy.te [hunk 2]
```diff
@@ -504,6 +503,9 @@ userdom_delete_user_tmp_files(snappy_mount_t)
 files_manage_generic_tmp_dirs(snappy_mount_t)
 files_manage_generic_tmp_files(snappy_mount_t)
 
+# Allow snap-{update,discard}-ns to mount on /etc/
```
Comment (olivercalder) [inline data/selinux/snappy.te:506]: Where are you getting the `snap-{update,discard}-ns` come from?

Comment (Meulengracht) [reply data/selinux/snappy.te:506]: As discussed, snap-confine uses snap-update-ns to actually do the mounts

**Pull request merged**